From 9db646973c7814ad2bfa5c3ba1d1fc6e5cf61d7d Mon Sep 17 00:00:00 2001
From: Karthikeyan Kathirvel <quic_kathirve@quicinc.com>
Date: Tue, 6 Feb 2024 17:16:50 +0530
Subject: [PATCH] cfg80211: add beacon protection enable bit mask

Add beacon protection enable bitmask, which will be used to fetch the
beacon protection enablement from the beacon ies.

Signed-off-by: Karthikeyan Kathirvel <quic_kathirve@quicinc.com>
---
 include/linux/ieee80211.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/ieee80211.h b/include/linux/ieee80211.h
index fb8f7cd..a9b7626 100644
--- a/include/linux/ieee80211.h
+++ b/include/linux/ieee80211.h
@@ -3838,6 +3838,9 @@ enum ieee80211_tdls_actioncode {
 /* Defines support for enhanced multi-bssid advertisement*/
 #define WLAN_EXT_CAPA11_EMA_SUPPORT	BIT(3)
 
+/* Enable Beacon Protection */
+#define WLAN_EXT_CAPA11_BCN_PROTECT	BIT(4)
+
 /* TDLS specific payload type in the LLC/SNAP header */
 #define WLAN_TDLS_SNAP_RFTYPE	0x2
 
-- 
2.34.1

